src/server/google-cloud.md

```diff
@@ -47,14 +56,14 @@ combined with Dart's Docker images.
 For more information, see the Compute Engine documentation for
 [using software containers][ce].
 
-## Kubernetes
+### Kubernetes
 
 To run Dart on clusters of Compute Engine instances,
 use Google Kubernetes Engine (GKE).
 
 For more information, see the [GKE overview][].
 
-## App Engine
+### App Engine
 
 [App Engine][] support for Dart is incomplete,
```

Comment:
Do we want to be even more specific here? E.g.:

"... is incomplete, and requires using AppEngine's flex environment, which will not automatically [scale to zero](https://cloud.google.com/run/docs/about-instance-autoscaling), so we..."

Reply:
PTAL